infra/base-images/base-runner/run_fuzzer

```diff
@@ -24,14 +24,16 @@ cd $OUT
 FUZZER=$1
 shift
 
-rm -rf /tmp/input/ && mkdir /tmp/input/
+CORPUS_DIR="/tmp/${FUZZER}_corpus"
+
+rm -rf ${CORPUS_DIR} && mkdir ${CORPUS_DIR}/
 
 CORPUS=
 SEED_CORPUS="${FUZZER}_seed_corpus.zip"
 if [ -f $SEED_CORPUS ]; then
   echo "Using seed corpus: $SEED_CORPUS"
-  unzip -d /tmp/input/ $SEED_CORPUS > /dev/null
-  CORPUS=/tmp/input
+  unzip -d ${CORPUS_DIR}/ $SEED_CORPUS > /dev/null
+  CORPUS=${CORPUS_DIR}
```

Comment:
Can we remove use of CORPUS, and just use CORPUS_DIR instead. I think even passing an empty dir instead of "" should be ok. It makes the code cleaner ? E.g. current code already has issue like hongfuzz does echo input > $CORPUS/input after a if -z $CORPUS condition.

Reply:
+1, removed `$CORPUS` completely. Done.